autogen.sh

```diff
@@ -8,6 +8,11 @@ for t in autoreconf aclocal pkg-config autoconf automake; do
 	if type $t; then
 		echo '------------------------------------------------------------------'
 		$t --version
+	else
+		echo '!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!'
+		echo "Please, install $t or reivse "'$PATH' 1>&2
```

Comment:
```suggestion
		echo "Please, install $t or revise "'$PATH' 1>&2
```

Reply:
I don't think we should show any warnings here. these warnings make me more confused. The error messages from autoreconf command is good engough.

Reply:
See #3845.